core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20170219_method_config_versions.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="method_config_version_column" author="ansingh" logicalFilePath="dummy">
+        <addColumn tableName="METHOD_CONFIG">
+            <column name="METHOD_CONFIG_VERSION" type="INT(11)" defaultValue="1" />
```

Comment:
Actually we do. In the rawls db, there is either INT(11) or BIGINT(20). For example in entity attribute:
<img width="175" alt="screen shot 2017-02-23 at 11 10 24 am" src="https://cloud.githubusercontent.com/assets/17534065/23267423/bd10ed0e-f9b8-11e6-9456-a6ac98e6e482.png">